todos/src/components/Todos.js

```diff
@@ -48,13 +56,23 @@ function Todos() {
               completedDate={elem.completedDate}
               onEdit={elem.onEdit}
             />
+          );
         })}
       </div>
+      {loadingState && (
+        <img className="spinner" src={spinner} alt="Loging"></img>
+      )}
       {showEmptyListIcon && <EmptyTaskList />}
-      {lessThanListLength ? (
+      {searchValue.length === 0 ? (
+        lessThanFilteredTodos ? (
+          <LoadMoreBtn type={LOAD_MORE} />
```

Comment:
As we are using the same button for two different purpose, we should name it more generally. 
E.G: `PaginatorButton`